NuRadioMC/examples/08_RNO_G_trigger_simulation/generate_data_set.py

```diff
@@ -0,0 +1,349 @@
+#!/bin/env python3
```

Comment:
what is the purpose of this script? Compared to `simulate.py` does it only add the two triggers and allows to use the NuRadioMCRunner? Can we not merge the two scripts into one (maybe we can have an second script which takes the simulation as defined in the first script but uses the runners?)

Reply:
@ryankrebs016 I still think we should merge the two scripts.